04_MMU/error-codes.yaml

```diff
@@ -352,7 +352,7 @@ Errors:
 
 - code: "04900"
   title: "UNKNOWN ERROR"
-  text: "Unexpected error occurred."
+  text: "Unexpected error occurred. If the issue persists, contact support."
```

Comment:
On MK3/S/+
It would not look nice as it will be split into 2 screens and the second one only contains the ResetMMU selection.
```
 |01234567890123456789
1|Unexpected error
2|occurred. If the
3|issue persists,
4|contact support.

1|
2|
3|
4|>ResetMMU          w
```

Reply:
not to be applied to 8bit